web/packages/teleport/src/Account/Account.test.tsx

```diff
@@ -47,7 +47,6 @@ describe('passkey + mfa button state', () => {
     ${'on'}       | ${false} | ${false}  | ${true}
     ${'optional'} | ${true}  | ${true}   | ${true}
     ${'optional'} | ${false} | ${false}  | ${true}
-    ${'otp'}      | ${true}  | ${false}  | ${true}
```

Comment:
Why was this test case removed?

Reply:
@zmb3 It got broken when I removed a redundant check, but it's also impossible in real life: the OTP option [implies](https://github.com/gravitational/teleport/blob/bl-nero/new-auth-device-dialog-5/api/constants/constants.go#L221) passwordless being disabled. The auth server refuses to start if these options are in conflict. Similarly, we don't check what happens if 2fa is set to "off" and passwordless is set to true.

If you still think it's good to check these seemingly impossible conditions, I can bring back this case (and extend the "off" case) to verify that we always enable passwordless when the backend tells us so, regardless of the MFA setting itself.

Reply:
(On the second thought, I just went on and included these anyway.)